crocket/src/main/java/com/crocket/Ball.java

```diff
@@ -42,14 +42,12 @@ public void startBall(double cosinus, double sinus, double power){
         setxVelocity(newXVelocity);
         setyVelocity(newYVelocity);
     }
-    
+
     public void move(){
         double xPosition = getxPosition();
         double yPosition = getyPosition();
         this.setxPosition((xPosition+xVelocity));
         this.setyPosition((yPosition+yVelocity));
         
-        getHitbox().setxPosition(getxPosition());
```

Comment:
Add these back, hould be overlooked in another branch/commit

Reply:
These are needed